packages/libsoup2.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Libsoup2 < Package
```

Comment:
Can we just update the current libsoup package with this version?  Not sure it makes sense to have 2 major versions of the same package.

Reply:
This is the older version. current libsoup package has the newer version.

Debian has it as libsoup-2.4: https://packages.debian.org/search?searchon=sourcenames&keywords=libsoup

Reply:
Do you understand why I'm so concerned about 2 different libsoup versions?  This can create a conflict between the files if both packages are installed at the same time.  The last one in wins.

Reply:
The libaries are versioned.
in libsoup2:
```
/usr/local/lib64/libsoup-2.4.so
/usr/local/lib64/libsoup-2.4.so.1
/usr/local/lib64/libsoup-2.4.so.1.11.0
/usr/local/lib64/libsoup-gnome-2.4.so
/usr/local/lib64/libsoup-gnome-2.4.so.1
/usr/local/lib64/libsoup-gnome-2.4.so.1.11.0
/usr/local/lib64/pkgconfig/libsoup-2.4.pc
/usr/local/lib64/pkgconfig/libsoup-gnome-2.4.pc
```
libsoup:
```
/usr/local/lib64/girepository-1.0/Soup-3.0.typelib
/usr/local/lib64/libsoup-3.0.so
/usr/local/lib64/libsoup-3.0.so.0
/usr/local/lib64/libsoup-3.0.so.0.0.0
/usr/local/lib64/pkgconfig/libsoup-3.0.pc
/usr/local/share/gir-1.0/Soup-3.0.gir
```

Reply:
Oh, so `2.99` is really version 3?  Weird.

Reply:
But let me add a dependency on the other!

Reply:
Is every other file in these packages different from the other?  I would hate to have any conflicts.

Reply:
Looks like they don't conflict:
```
[1;31mlibsoup2[0m: libsoup is an HTTP client/server library for GNOME.
/usr/local/include/libsoup-2.4/libsoup/soup-address.h
/usr/local/include/libsoup-2.4/libsoup/soup-auth-domain-basic.h
/usr/local/include/libsoup-2.4/libsoup/soup-auth-domain-digest.h
/usr/local/include/libsoup-2.4/libsoup/soup-auth-domain.h
/usr/local/include/libsoup-2.4/libsoup/soup-auth.h
/usr/local/include/libsoup-2.4/libsoup/soup-auth-manager.h
/usr/local/include/libsoup-2.4/libsoup/soup-autocleanups.h
/usr/local/include/libsoup-2.4/libsoup/soup-cache.h
/usr/local/include/libsoup-2.4/libsoup/soup-content-decoder.h
/usr/local/include/libsoup-2.4/libsoup/soup-content-sniffer.h
/usr/local/include/libsoup-2.4/libsoup/soup-cookie.h
/usr/local/include/libsoup-2.4/libsoup/soup-cookie-jar-db.h
/usr/local/include/libsoup-2.4/libsoup/soup-cookie-jar.h
/usr/local/include/libsoup-2.4/libsoup/soup-cookie-jar-text.h
/usr/local/include/libsoup-2.4/libsoup/soup-date.h
/usr/local/include/libsoup-2.4/libsoup/soup-enum-types.h
/usr/local/include/libsoup-2.4/libsoup/soup-form.h
/usr/local/include/libsoup-2.4/libsoup/soup.h
/usr/local/include/libsoup-2.4/libsoup/soup-headers.h
/usr/local/include/libsoup-2.4/libsoup/soup-hsts-enforcer-db.h
/usr/local/include/libsoup-2.4/libsoup/soup-hsts-enforcer.h
/usr/local/include/libsoup-2.4/libsoup/soup-hsts-policy.h
/usr/local/include/libsoup-2.4/libsoup/soup-logger.h
/usr/local/include/libsoup-2.4/libsoup/soup-message-body.h
/usr/local/include/libsoup-2.4/libsoup/soup-message.h
/usr/local/include/libsoup-2.4/libsoup/soup-message-headers.h
/usr/local/include/libsoup-2.4/libsoup/soup-method.h
/usr/local/include/libsoup-2.4/libsoup/soup-misc.h
/usr/local/include/libsoup-2.4/libsoup/soup-multipart.h
/usr/local/include/libsoup-2.4/libsoup/soup-multipart-input-stream.h
/usr/local/include/libsoup-2.4/libsoup/soup-password-manager.h
/usr/local/include/libsoup-2.4/libsoup/soup-portability.h
/usr/local/include/libsoup-2.4/libsoup/soup-proxy-resolver-default.h
/usr/local/include/libsoup-2.4/libsoup/soup-proxy-resolver.h
/usr/local/include/libsoup-2.4/libsoup/soup-proxy-uri-resolver.h
/usr/local/include/libsoup-2.4/libsoup/soup-request-data.h
/usr/local/include/libsoup-2.4/libsoup/soup-requester.h
/usr/local/include/libsoup-2.4/libsoup/soup-request-file.h
/usr/local/include/libsoup-2.4/libsoup/soup-request.h
/usr/local/include/libsoup-2.4/libsoup/soup-request-http.h
/usr/local/include/libsoup-2.4/libsoup/soup-server.h
/usr/local/include/libsoup-2.4/libsoup/soup-session-async.h
/usr/local/include/libsoup-2.4/libsoup/soup-session-feature.h
/usr/local/include/libsoup-2.4/libsoup/soup-session.h
/usr/local/include/libsoup-2.4/libsoup/soup-session-sync.h
/usr/local/include/libsoup-2.4/libsoup/soup-socket.h
/usr/local/include/libsoup-2.4/libsoup/soup-status.h
/usr/local/include/libsoup-2.4/libsoup/soup-tld.h
/usr/local/include/libsoup-2.4/libsoup/soup-types.h
/usr/local/include/libsoup-2.4/libsoup/soup-uri.h
/usr/local/include/libsoup-2.4/libsoup/soup-value-utils.h
/usr/local/include/libsoup-2.4/libsoup/soup-version.h
/usr/local/include/libsoup-2.4/libsoup/soup-websocket-connection.h
/usr/local/include/libsoup-2.4/libsoup/soup-websocket-extension-deflate.h
/usr/local/include/libsoup-2.4/libsoup/soup-websocket-extension.h
/usr/local/include/libsoup-2.4/libsoup/soup-websocket-extension-manager.h
/usr/local/include/libsoup-2.4/libsoup/soup-websocket.h
/usr/local/include/libsoup-2.4/libsoup/soup-xmlrpc.h
/usr/local/include/libsoup-2.4/libsoup/soup-xmlrpc-old.h
/usr/local/include/libsoup-gnome-2.4/libsoup/soup-cookie-jar-sqlite.h
/usr/local/include/libsoup-gnome-2.4/libsoup/soup-gnome-features.h
/usr/local/include/libsoup-gnome-2.4/libsoup/soup-gnome.h
/usr/local/lib64/girepository-1.0/Soup-2.4.typelib
/usr/local/lib64/girepository-1.0/SoupGNOME-2.4.typelib
/usr/local/lib64/libsoup-2.4.so
/usr/local/lib64/libsoup-2.4.so.1
/usr/local/lib64/libsoup-2.4.so.1.11.0
/usr/local/lib64/libsoup-gnome-2.4.so
/usr/local/lib64/libsoup-gnome-2.4.so.1
/usr/local/lib64/libsoup-gnome-2.4.so.1.11.0
/usr/local/lib64/pkgconfig/libsoup-2.4.pc
/usr/local/lib64/pkgconfig/libsoup-gnome-2.4.pc
/usr/local/share/gir-1.0/Soup-2.4.gir
/usr/local/share/gir-1.0/SoupGNOME-2.4.gir
/usr/local/share/locale/an/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/as/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/be/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/bg/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/bn_IN/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/bs/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ca/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ca@valencia/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/cs/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/da/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/de/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/el/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/en_GB/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/eo/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/es/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/et/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/eu/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/fa/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/fi/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/fr/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/fur/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/gd/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/gl/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/gu/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/he/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/hi/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/hr/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/hu/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/id/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/it/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ja/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/kn/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ko/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/lt/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/lv/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ml/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/mr/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ms/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/nb/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ne/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/nl/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/oc/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/or/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/pa/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/pl/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/pt_BR/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/pt/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ro/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ru/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/sk/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/sl/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/sr@latin/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/sr/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/sv/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ta/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/te/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/tg/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/th/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/tr/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/ug/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/uk/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/uz@cyrillic/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/vi/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/zh_CN/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/zh_HK/LC_MESSAGES/libsoup.mo
/usr/local/share/locale/zh_TW/LC_MESSAGES/libsoup.mo
/usr/local/share/vala/vapi/libsoup-2.4.deps
/usr/local/share/vala/vapi/libsoup-2.4.vapi
[1;32mTotal found: 142[0m
[1;32mDisk usage: 3.0MB
[0m
[1;32mlibsoup[0m: libsoup is an HTTP client/server library for GNOME.
/usr/local/include/libsoup-3.0/libsoup/soup-auth-domain-basic.h
/usr/local/include/libsoup-3.0/libsoup/soup-auth-domain-digest.h
/usr/local/include/libsoup-3.0/libsoup/soup-auth-domain.h
/usr/local/include/libsoup-3.0/libsoup/soup-auth.h
/usr/local/include/libsoup-3.0/libsoup/soup-auth-manager.h
/usr/local/include/libsoup-3.0/libsoup/soup-cache.h
/usr/local/include/libsoup-3.0/libsoup/soup-content-decoder.h
/usr/local/include/libsoup-3.0/libsoup/soup-content-sniffer.h
/usr/local/include/libsoup-3.0/libsoup/soup-cookie.h
/usr/local/include/libsoup-3.0/libsoup/soup-cookie-jar-db.h
/usr/local/include/libsoup-3.0/libsoup/soup-cookie-jar.h
/usr/local/include/libsoup-3.0/libsoup/soup-cookie-jar-text.h
/usr/local/include/libsoup-3.0/libsoup/soup-date-utils.h
/usr/local/include/libsoup-3.0/libsoup/soup-enum-types.h
/usr/local/include/libsoup-3.0/libsoup/soup-form.h
/usr/local/include/libsoup-3.0/libsoup/soup.h
/usr/local/include/libsoup-3.0/libsoup/soup-headers.h
/usr/local/include/libsoup-3.0/libsoup/soup-hsts-enforcer-db.h
/usr/local/include/libsoup-3.0/libsoup/soup-hsts-enforcer.h
/usr/local/include/libsoup-3.0/libsoup/soup-hsts-policy.h
/usr/local/include/libsoup-3.0/libsoup/soup-logger.h
/usr/local/include/libsoup-3.0/libsoup/soup-message-body.h
/usr/local/include/libsoup-3.0/libsoup/soup-message.h
/usr/local/include/libsoup-3.0/libsoup/soup-message-headers.h
/usr/local/include/libsoup-3.0/libsoup/soup-method.h
/usr/local/include/libsoup-3.0/libsoup/soup-multipart.h
/usr/local/include/libsoup-3.0/libsoup/soup-multipart-input-stream.h
/usr/local/include/libsoup-3.0/libsoup/soup-server.h
/usr/local/include/libsoup-3.0/libsoup/soup-server-message.h
/usr/local/include/libsoup-3.0/libsoup/soup-session-feature.h
/usr/local/include/libsoup-3.0/libsoup/soup-session.h
/usr/local/include/libsoup-3.0/libsoup/soup-status.h
/usr/local/include/libsoup-3.0/libsoup/soup-tld.h
/usr/local/include/libsoup-3.0/libsoup/soup-types.h
/usr/local/include/libsoup-3.0/libsoup/soup-uri-utils.h
/usr/local/include/libsoup-3.0/libsoup/soup-version.h
/usr/local/include/libsoup-3.0/libsoup/soup-websocket-connection.h
/usr/local/include/libsoup-3.0/libsoup/soup-websocket-extension-deflate.h
/usr/local/include/libsoup-3.0/libsoup/soup-websocket-extension.h
/usr/local/include/libsoup-3.0/libsoup/soup-websocket-extension-manager.h
/usr/local/include/libsoup-3.0/libsoup/soup-websocket.h
/usr/local/lib64/girepository-1.0/Soup-3.0.typelib
/usr/local/lib64/libsoup-3.0.so
/usr/local/lib64/libsoup-3.0.so.0
/usr/local/lib64/libsoup-3.0.so.0.0.0
/usr/local/lib64/pkgconfig/libsoup-3.0.pc
/usr/local/share/gir-1.0/Soup-3.0.gir
/usr/local/share/locale/an/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/as/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/be/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/bg/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/bn_IN/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/bs/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ca/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ca@valencia/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/cs/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/da/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/de/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/el/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/en_GB/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/eo/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/es/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/et/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/eu/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/fa/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/fi/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/fr/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/fur/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/gd/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/gl/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/gu/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/he/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/hi/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/hr/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/hu/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/id/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/it/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ja/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/kn/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ko/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/lt/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/lv/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ml/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/mr/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ms/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/nb/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ne/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/nl/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/oc/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/or/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/pa/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/pl/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/pt_BR/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/pt/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ro/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ru/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/sk/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/sl/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/sr@latin/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/sr/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/sv/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ta/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/te/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/tg/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/th/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/tr/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/ug/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/uk/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/uz@cyrillic/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/vi/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/zh_CN/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/zh_HK/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/locale/zh_TW/LC_MESSAGES/libsoup-3.0.mo
/usr/local/share/vala/vapi/libsoup-3.0.deps
/usr/local/share/vala/vapi/libsoup-3.0.vapi
[1;32mTotal found: 115[0m
[1;32mDisk usage: 2.0MB
[0m
```